### ISSUE TITLE
100% GPU usage on ultra settings. High GPU usage in general. 

### ISSUE THREAD

**Issue by JamieBurn99** [PrehistoricKingdom/feedback#179]
Turning down graphical settings fixes this issue but my GPU gets close to overheating on ultra. 
I have a GTX 1660 TI and it doesn't go above 50 degrees on any other game so I know this isn't a problem on my end.

**Comment by diddle783** [comment]
mine is also getting warm 68 degrees. I'm having a gforce gtx 2070 super and other games never hit that heat so far

**Comment by JamieBurn99** [comment]
Yeah it's just an optimisation issue is all it is. Nothing major but turning down the graphical settings helps a bit. Mine hits around 70 then stops but this is the highest my GPU ever goes :)

**Comment by crytivo-dave** [comment]
70c is actually cool for a rtx 2000 series. Those cards are meant to run under heavy load at 80-85c. You also actually want a game at 100% GPU because it means it is being fully used.
